mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -248,12 +269,38 @@ class GBTClassificationModel private[ml](
     if (prediction > 0.0) 1.0 else 0.0
   }
 
+  override protected def predictRaw(features: Vector): Vector = {
```

Comment:
In logistic regression we had previously overridden some of the methods in probabilistic classifier since we were only dealing with two classes, which makes those methods a bit faster (hard to say how much). We can do it here for now, but I'd be slightly in favor of _not_ doing it since I'm not sure how much we gain from it and it makes the code harder to follow. Thoughts?